build.gradle

```diff
@@ -127,3 +129,7 @@ ext {
 
   accompanistFlowLayout = "com.google.accompanist:accompanist-flowlayout:0.20.2"
 }
+
+tasks.dokkaHtmlMultiModule.configure {
```

Comment:
this combines all dokka documents from separate modules into one (still separated by module, but there's an index site with a link to all)